py/plugins/dev-local-vector-store/src/genkit/plugins/dev_local_vector_store/constant.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2025 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+from genkit.ai.retriever import CommonRetrieverOptionsSchema
+from genkit.core.typing import DocumentData, Embedding
+from pydantic import BaseModel
+
+
+class Params(BaseModel):
+    index_name: str
+    embedder: str
+    embedder_options: CommonRetrieverOptionsSchema
```

Comment:
this should be optional

Reply:
Also, this should be embedder options (not retriever options). (note that we should resolve whether  we need these common options first)